pages/_app.tsx

```diff
@@ -4,11 +4,29 @@ import Head from 'next/head';
 
 import { Header, Footer } from '@components';
 import { Maintenance } from 'components/Maintenance';
+import Script from 'next/script';
 
 // TODO should be an ENV var
 const isMaintenanceMode = false;
 
+const envName = process.env.NEXT_PUBLIC_ENV_NAME;
+
+const getAnalyticsScriptPath = (envName?: string) => {
+  switch (envName) {
+    case 'dev':
```

Comment:
https://stackoverflow.com/questions/13207927/switch-statement-for-multiple-cases-in-javascript

Reply:
Fair enough 😄